README.md

```diff
@@ -101,12 +116,31 @@ git commit --amend                      # Re-do the commit message of the previo
 - or for instance Notepad++:
 `git config --global core.editor "'C:/Program Files/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`
 
+# Re/moving files under version control
+git rm <path/to/the/file>                 # remove file and stage the change to be committed.
+git mv <source/file> <destination/file>   # move/rename file and stage the change to be committed.  
```

Comment:
This part is fine, but should probably be further down in cheat sheet.

Reply:
Moved it down.